### PR TITLE
Update dashboard FAQ

### PR DESCRIPTION
**Pull request by shhdgit**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Added a new FAQ for TiDB Dashboard

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- Other reference link(s): [tidb-dashboard#902](https://github.com/pingcap/tidb-dashboard/issues/902)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
This PR is translation from https://github.com/pingcap/docs-cn/pull/6178.

**Comment by shhdgit** [comment]
Updated, PTAL.

**Comment by TomShawn** [comment]
@shhdgit Please help confirm whether the PR change applies to v5.1. Thanks~

**Comment by shhdgit** [comment]
Yes, v5.1 can also be applied.

**Comment by TomShawn** [comment]
@breeswish PTAL again, thanks~

File: dashboard/dashboard-faq.md [hunk 1]
```diff
@@ -63,10 +63,12 @@ The possible reason is that you have enabled the `prepared-plan-cache` feature o
 
 If the `unknown field` error appears on the Slow Queries page after the cluster upgrade, that is a compatibility issue caused by the difference between TiDB Dashboard server fields (which may be updated) and user preferences fields (which are in the browser cache). This issue has been fixed in v4.0.14/v5.0.3 and later, older versions can be resolved by clearing the browser Local Storage.
 
+1. Open Dashboard page
```
Comment (breeswish) [inline dashboard/dashboard-faq.md:66]:
```suggestion
1. Open TiDB Dashboard page
```

File: dashboard/dashboard-faq.md [hunk 2]
```diff
@@ -63,10 +63,12 @@ The possible reason is that you have enabled the `prepared-plan-cache` feature o
...
+
 1. Click **Customize and control Google Chrome** and then select **More Tools > Developer Tools**.
 
 ![Opening DevTools from Chrome's main menu](/media/dashboard/dashboard-faq-devtools.png)
 
-2. Select the **Application** panel, expand the **Local Storage** menu and select the **Dashboard page domain**, click the **Clear All** button.
+1. Select the **Application** panel, expand the **Local Storage** menu and select the **Dashboard page domain**, click the **Clear All** button.
```
Comment (breeswish) [inline dashboard/dashboard-faq.md]:
```suggestion
1. Select the **Application** panel, expand the **Local Storage** menu and select the **TiDB Dashboard page domain**, click the **Clear All** button.
```

**Review by breeswish — APPROVED** [review]

**Comment by breeswish** [comment]
@TomShawn PTAL, thanks!

File: dashboard/dashboard-faq.md [hunk 3]
```diff
@@ -58,3 +58,17 @@ If your deployment tool is TiUP, take the following steps to solve this problem.
 ### An `invalid connection` error is shown in **Top SQL Statements** and **Recent Slow Queries** on the Overview page
 
 The possible reason is that you have enabled the `prepared-plan-cache` feature of TiDB. As an experimental feature, when enabled, `prepared-plan-cache` might not function properly in specific TiDB versions, which could cause this problem in TiDB Dashboard (and other applications). You can disable `prepared-plan-cache` by updating [TiDB Configuration file](/tidb-configuration-file.md#prepared-plan-cache) to solve this problem.
+
+### An `unknown field` error is shown in **Slow Queries** page
+
+If the `unknown field` error appears on the Slow Queries page after the cluster upgrade, that is a compatibility issue caused by the difference between TiDB Dashboard server fields (which may be updated) and user preferences fields (which are in the browser cache). This issue has been fixed in v4.0.14/v5.0.3 and later, older versions can be resolved by clearing the browser Local Storage.
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:64]: @shhdgit @breeswish This PR is based on release-4.0 and will be **first** merged into the v4.0 documentation. If it applies to v4.0.14, I am afraid the merge will be no earlier than the v4.0.14 release.

Comment (breeswish) [reply dashboard/dashboard-faq.md:64]: Oh, good catch!! @shhdgit Could you propose a PR based on master instead?

Comment (shhdgit) [reply dashboard/dashboard-faq.md:64]: Sure.

**Comment by TomShawn** [comment]
@shhdgit Since this PR content is migrated to https://github.com/pingcap/docs/pull/5895, shall we close this one?

**Comment by shhdgit** [comment]
Yes.

**Pull request closed without merging**